app/invoice.rb

```diff
@@ -1,5 +1,6 @@
 class Invoice < Sequel::Model
   AlreadyFinalized = Class.new(StandardError)
+  ProcessingError = Class.new(StandardError)
```

Comment:
Do we still need this now? It's not used at the server side anymore. If you want to keep it we should probably move it to the client.

Reply:
Should have cleaned up house better, apologies…